surface.js

```diff
@@ -66,10 +66,12 @@ function SurfacePickResult(position, index, uv, level, dataCoordinate) {
   this.dataCoordinate = dataCoordinate
 }
 
+var N_COLORS = 265
+
 function genColormap(name) {
   var x = pack([colormap({
     colormap: name,
-    nshades: 256,
+    nshades: N_COLORS,
     format: 'rgba'
   }).map(function(c) {
     return [c[0], c[1], c[2], 255*c[3]]
```

Comment:
I don't think they were meant to match.

Replacing `255` with `N_COLORS` does change the colormap slightly.